.env.dist

```
@@ -0,0 +1,25 @@
+###> symfony/framework-bundle ###
+APP_ENV=dev
+URL=local.appbuild.com
+JAWSDB_URL=mysql://appbuild_usr:appbuild_pwd@local.appbuild-db.com/appbuild_db
```

```suggestion
DB_URL=mysql://appbuild_usr:appbuild_pwd@local.appbuild-db.com/appbuild_db
```

we won't force this project users to use Heroku